web/gds-admin-ui/src/pages/app/lists/index.js

```diff
@@ -50,10 +51,7 @@ const StatusColumn = ({ row }) => {
     return (
         <React.Fragment>
             <span
-                className={classNames('badge', {
-                    'bg-success': row.original.verification_status === Status.VERIFIED,
-                    'bg-warning': row.original.verification_status === Status.SUBMITTED || row.original.verification_status === Status.PENDING_REVIEW,
-                })}>
+                className={classNames('badge', getStatusClassName(row.original.verification_status))}>
```

Comment:
Before, was the badge uncolored and the text simply white, which is why we could not see it?